cli/src/commands/get.js

```diff
@@ -33,13 +34,18 @@ ${bold('EXAMPLES')}
         ${cyan('comfy get staging -t next --yml > config/staging.yaml')}
         ${dim('# Get the production configuration and set it as environment variables')}
         ${cyan('comfy get production --envvars | source /dev/stdin')}
+
+        ${dim('# Get only a field of the config')}
+        ${dim('config.json: { "admin": { "user": "Admin", "pass": "1234" } }')}
+        ${cyan('comfy get production admin.user')} // Admin
 `);
 };
 
 module.exports = (ui, modules) => function* get(rawOptions) {
     const { bold, green, red } = ui.colors;
     const options = minimist(rawOptions);
     const env = options._[0];
+    const selector = options._[1];
```

Comment:
Can't you use destructuring here ?

`const [env, selector] = options._`

Reply:
Yes I can, you're right!